src/main/java/chess/board/Board.java

```diff
@@ -1,62 +1,79 @@
 package chess.board;
 
-import chess.pieces.Pawn;
+import static chess.utils.StringUtils.appendNewLine;
+
+import chess.pieces.Color;
+import chess.pieces.Name;
+import chess.pieces.Piece;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
-import java.util.stream.Collectors;
+import java.util.Objects;
 
 public class Board {
-    private final List<Pawn> allPawns = new ArrayList<>();
-    private final List<Pawn> whitePawns = new ArrayList<>();
-    private final List<Pawn> blackPawns = new ArrayList<>();
+    public static final int COLUMN_AND_ROW_SIZE = 8;
+    private static final int BLACK_INITIAL_OTHERS_ROW = 0;
+    private static final int BLACK_INITIAL_PAWNS_ROW = 1;
+    private static final int WHITE_INITIAL_OTHERS_ROW = 7;
+    private static final int WHITE_INITIAL_PAWNS_ROW = 6;
 
+    public static final String EMPTY_SPACE = ".";
+    public static final String EMPTY_STRING = "";
+    private final Piece[][] pieces = new Piece[COLUMN_AND_ROW_SIZE][COLUMN_AND_ROW_SIZE];
 
     public void initialize() {
-        for (int i = 0; i < 8; i++) {
-            add(new Pawn());
-            add(new Pawn(Pawn.BLACK_COLOR));
-        }
-    }
+        List<Name> otherNames = makeOtherNames();
```

Comment:
변수랑 메소드 이름이 살짝 맘에 걸리는데 더 좋은 이름이 저도 생각나지는 않네요 ㅜㅜ. 

```suggestion
        List<Name> allNamesExceptPawns = otherPiecesNames();
```
아 몰라요 ㅋㅋ 없던 걸로...